src/controllers/register.js

```diff
@@ -0,0 +1,11 @@
+export const registerRoute = {
```

Comment:
I added this as i was not seeing this in your commit @AakrritiJain , build was failing due to this, you can push your changes too.